hack/jenkins/benchmark-dockerized.sh

```diff
@@ -52,3 +49,5 @@ cd /go/src/k8s.io/kubernetes
 # Run the benchmark tests and pretty-print the results into a separate file.
 make test-integration WHAT="$*" KUBE_TEST_ARGS="-run='XXX' -bench=. -benchmem" \
   | tee >(prettybench -no-passthrough > ${ARTIFACTS_DIR}/BenchmarkResults.txt)
+
+sleep 100
```

Comment:
1. Why do you need this?

2. Remove this funny character at the end of line :)